parquet-avro/src/test/java/org/apache/parquet/avro/TestReadVariant.java

```diff
@@ -18,16 +18,25 @@
  */
 package org.apache.parquet.avro;
 
-import static org.apache.parquet.avro.AvroTestUtil.*;
-import static org.junit.Assert.*;
```

Comment:
Avoid * imports for Java style. It's easier to understand where symbols come from for reviews this way.